javascript-bank/account.js

```diff
@@ -0,0 +1,47 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var transactionsList = this.transactions;
+  var newTransaction = new Transaction('deposit', amount);
+
+  if (amount > 0) {
+    transactionsList.push(newTransaction);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  var transactionsList = this.transactions;
+  var newTransaction = new Transaction('withdraw', amount);
+  if (amount > 0) {
+    transactionsList.push(newTransaction);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  var transactionsList = this.transactions;
+  var balance = 0;
+  if (transactionsList.length === 0) {
+    return 0;
+  }
```

Comment:
This can be removed. If there are no entries in the array, the loop will never start, so you are left with the same result.